server/utils/populate-all.js

```diff
@@ -11,25 +11,31 @@ const _ = require('lodash')
  * @returns a populate object with all components, nested components,
```

Comment:
Also mismatch between implementation and documentation
```suggestion
 * @param {object} options - options for recursion and population
 * @param {number} options.maxDepth - maximum depth for recursive population, defaults to 10
 * @param {boolean} options.populateMedia - whether to include media, defaults to false
 * @returns a populate object with all components, nested components,
```